CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # @digitalbazaar/vc ChangeLog
 
+## 6.0.0 - 2023-TBD
+
+### Removed
+- **BREAKING**: Remove `odrl` and `vc examples` contexts from `./lib/contexts`.
```

Comment:
Should note contexts are now available in `@digitalbazaar/odrl-context` and `@digitalbazaar/credentials-examples-context`. Just in case anyone had been using them from here.

Reply:
Added [805a725](https://github.com/digitalbazaar/vc/pull/117/commits/805a72540ce8524b6fa440b57086630002e25e06)